actix-http/CHANGES.md

```diff
@@ -365,6 +365,8 @@
 [#2585]: https://github.com/actix/actix-web/pull/2585
 [#2587]: https://github.com/actix/actix-web/pull/2587
 
+### Fixed
+- Encode correctly camel case header with n+2 hyphens [#2674]
```

Comment:
this should be added under "unreleased"

Reply:
How did I end up there... I have no clue. I will update as soon as I'm back home